examples/tutorials/color-figure.py

```diff
@@ -0,0 +1,95 @@
+"""
+Adding a color pallete to a figure
```

Comment:
Typo.
```suggestion
Adding a color palette to a figure
```

Reply:
Much like I discovered as soon as I hung up my AGU poster, no amount of drafts will stop typos

Reply:
@willschlitzer Actually there are a lot of "pallete" typos throughout the whole file. You need to correct it, and I will find some time reviewing it.

Reply:
Sorry about that; just pushed a change that fixed the typo throughout